src/index.html

```diff
@@ -7,8 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          class="header__logo"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a
+              href="./"
+              class="header__link is-active"
+            >apple
+            </a>
```

Comment:
```suggestion
              class="header__link is-active"
            >
             Apple
            </a>
```
![image](https://user-images.githubusercontent.com/64437769/142387146-4d26c68d-a1d5-41e5-ad70-1e16bea23dbd.png)